FIAT/reference_element.py

```diff
@@ -1131,6 +1227,70 @@ def __ge__(self, other):
     def __le__(self, other):
         return self.compare(operator.le, other)
 
+    def extract_extrinsic_orientation(self, o):
+        """Extract extrinsic orientation.
+
+        Parameters
+        ----------
+        o : object
```

Comment:
I think we need a base class for `o` that tells us how `//` and `%` are supposed to behave.

Reply:
Added `Orientation` class.